.github/workflows/main.yml

```diff
@@ -2,8 +2,67 @@ name: CI
 on: [push, pull_request]
 
 jobs:
-  test:
-    name: Test
+  buildlibc:
+    name: Build libc
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+
+    - name: Install clang (Windows)
+      shell: bash
+      run: |
+          curl -fsSLO https://releases.llvm.org/8.0.0/LLVM-8.0.0-win64.exe
+          7z x LLVM-8.0.0-win64.exe -y -o"llvm"
+          echo "::add-path::$(pwd)/llvm/bin"
+          echo "::set-env name=WASM_AR::$(pwd)/llvm/bin/llvm-ar.exe"
```

Comment:
How do these echo commands to?  Do that perform some action?  Is github reading the stdout of the bash and doing something here?

Reply:
Yes, these echo commands tell GH actions to append to the path or set an environment variable. Azure had something that worked similarly but looked a lot more like line noise. https://help.github.com/en/actions/reference/development-tools-for-github-actions#set-an-environment-variable-set-env

Reply:
Oh wow.. thats gross.